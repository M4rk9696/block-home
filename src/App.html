<Quotes quote="{$quote}"/>
<div class="inline">
  <TileSet sites="{$sites}"/>
</div>
<ConfigButton/>
{#if $showConfigQuotes }
  <div transition:fade="{delay: 0, duration: 100}">
    <ConfigQuotes quotes="{$quotes}"/>
  </div>
{/if}
{#if $showConfig }
  <div transition:fade="{delay: 0, duration: 100}">
    <Config sites="{$sites}"/>
  </div>
{/if}
{#if $showQuickConfig}
  <div transition:fade="{delay: 0, duration: 140}">
    <QuickConfig site="{$quickConfigSite}"/>
  </div>
{/if}

<script>
  import { fade } from 'svelte-transitions';

  export default {
    components: {
      TileSet: './components/TileSet.html',
      Config: './components/Config.html',
      Quotes: './components/Quotes.html',
      ConfigButton: './components/ConfigButton.html',
      QuickConfig: './components/QuickConfig.html',
      ConfigQuotes: './components/ConfigQuotes.html'
    },
    transitions: { fade }
  };
</script>

<style>
</style>
