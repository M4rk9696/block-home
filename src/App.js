import React from 'react'
import TileSet from './components/TileSet'
import { SiteProvider } from './providers/SiteContext'
import { ConfigProvider } from './providers/ConfigContext'
import './global.css'

const App = () => {
  return (
    <ConfigProvider>
      <SiteProvider>
        <div className="inline">
          <TileSet />
        </div>
      </SiteProvider>
    </ConfigProvider>
  )
}

export default App
