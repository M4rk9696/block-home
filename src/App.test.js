import React from 'react'
import TestRenderer from 'react-test-renderer'
import App from './App'

describe('should render App', () => {
  it('should be able to mount the component', () => {
    const component = TestRenderer.create(<App />)

    expect(component.toJSON()).toMatchSnapshot()
  })
})
