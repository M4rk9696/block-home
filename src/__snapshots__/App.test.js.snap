// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App should be able to mount the component 1`] = `
<div>
  Hello World
</div>
`;
